### DISCUSSION TITLE
Feature Request: User sharing

### DISCUSSION THREAD

**Discussion by nothingmn** [Freika/dawarich#495] [Ideas]
For a family I would like to see users being able to share data with other users.  I would like to be able to choose which users I see on the maps, trips and other pages.

ps. Great job on this!
pps. Since there is no specific template/format for a feature request, I kept it minimal.

**Comment by tismofied** [comment]
I would love this feature as well.

**Comment by Freika** [comment]
@nothingmn @tismofied FYI 0.34.0 (RC branch, docker image `freikin/dawarich:rc`) release contains the Family feature, available for testing (image only built for AMD64 arch)

**Comment by Freika** [comment]
Let's continue discussion in #134 though
